cmd/protolock/main.go

```diff
@@ -132,7 +132,9 @@ func status(cfg *protolock.Config) {
 		os.Exit(1)
 	}
 
-	os.Exit(code)
+	if code != 0 {
+		os.Exit(code)
+	}
```

Comment:
the reason to even handle this, is because in the future we may want to use other exit codes (see #102) to indicate other failure modes which can be properly interpreted by users' automated integration systems